tests/integration/security/ca_custom_root/secure_naming_test.go

```diff
@@ -107,7 +107,7 @@ func TestSecureNaming(t *testing.T) {
 		Features("security.peer.secure-naming").
 		Run(func(t framework.TestContext) {
 			// TODO https://github.com/istio/istio/issues/32292
-			if t.Clusters().IsMulticluster() {
+			if t.Clusters().IsMulticluster() || t.Clusters().Default().Primary().IsExternalControlPlane() {
```

Comment:
Why does `IsMulticluster` not cover external control plane? external control plane is by definition multicluster, isn't it?

Reply:
No. `IsMulticluster` only refers to "mesh clusters" (it doesn't consider clusters outside the mesh). I'm currently trying to run these tests in a single cluster (config cluster) w external control plane topology and running into the same issue(s).

Reply:
That gives me an idea though. `t.AllClusters().IsMulticluster()` is a simpler check.